README.md

```diff
@@ -108,9 +108,13 @@ Returns a function for generating random numbers with a [Poisson distribution](h
 Returns the same type of function for generating random numbers but where the given random number generator *source* is used as the source of randomness instead of Math.random. The given random number generator must implement the same interface as Math.random and only return values in the range [0, 1). This is useful when a seeded random number generator is preferable to Math.random. For example:
 
 ```js
-var d3 = require("d3-random"),
-    seedrandom = require("seedrandom"),
-    random = d3.randomNormal.source(seedrandom("a22ebc7c488a3a47"))(0, 1);
+const d3 = require("d3-random"),
+    seed = 42,
+    random = d3.randomNormal.source(d3.randomLcg(seed))(0, 1);
 
-random(); // 0.9744193494813501
+random(); // -0.26990580687568544
 ```
+
+<a name="randomLcg" href="#randomLcg">#</a> d3.<b>randomLcg</b>(<i>[seed]</i>) · [Examples](https://observablehq.com/@fil/linear-congruential-generator)
+
+Returns a [Linear congruential generator](https://en.wikipedia.org/wiki/Linear_congruential_generator), seeded on the specified non-negative integer, which defaults to 0. The generator can be called repeatedly to obtain a pseudo-random sequence of values, well-distributed on the interval [0,1), and with a long period (up to 1 billion numbers). When run with the same seed, the sequence is guaranteed to be the same. Different seeds will usually result in different sequences, but no guarantee is made.
```

Comment:
This documentation implies that the seed must be a non-negative integer, but the implementation currently maps floating point values in (-1, 1) to integers. I like the idea of an initial mix, but I think it’d be better to document that a “good” seed is a random integer in [0, Math.pow(2, 32) - 1], and then simply error if the provided seed does not match these expectations.

```js
s = Math.floor(s);
if (!(0 <= s && s < m)) throw new RangeError("invalid seed");
```

Also, it’d probably be better to default to a random seed, rather than `s = 1`, no? Per above, a good default seed could be `s = Math.random() * m`.

Reply:
Or perhaps a more natural choice from the caller’s perspective is that the seed should be in [0, 1) and default to `s = Math.random()`. Then on initialization, we’d say:

```js
if (!(0 <= s && s < 1)) throw new RangeError("invalid seed");
s = Math.floor(m * s);
```

Reply:
OK—I like this second option, it's easy to explain.